File: training_js/32/python/main.py
import math

def round_it(n):
    string = (str(n).split("."))
    if len(string[0]) < len(string[1]):
        return math.ceil(n)
    elif len(string[0]) > len(string[1]):
        return math.floor(n)
    else:
        return round(n)

File: training_js/32/javascript/main.js
function roundIt(n){
  string = String(n).split(".");
  if (string[0].length < string[1].length){
    return Math.ceil(n);
  }else if (string[0].length > string[1].length){
    return Math.floor(n);
  }else {
    return Math.round(n);
  }
}
